README.md

```diff
@@ -502,6 +505,7 @@ Check out [this part](https://thundra.readme.io/docs/how-to-warmup) in our docs
 | THUNDRA_AGENT_LAMBDA_DEBUGGER_AUTH_TOKEN                            | string |                -                |
 | THUNDRA_AGENT_TRACE_INTEGRATIONS_HAPI_DISABLE                       |  bool  |              false              |
 | THUNDRA_AGENT_TRACE_INTEGRATIONS_KOA_DISABLE                        |  bool  |              false              |
+| THUNDRA_AGENT_TRACE_INTEGRATIONS_GOOGLE_PUBSUB_DISABLE              |  bool  |              false              |
```

Comment:
Can you also add `THUNDRA_AGENT_TRACE_INTEGRATIONS_GOOGLE_BIGQUERY_DISABLE` env var to disable Google BigQuery integration